### PR TITLE
[IGListIndexSetResult hasChanges]` should instead be a readonly property #402

### PR DESCRIPTION
**Pull request by AdamRobertsEF**
Changed method to read only property.

### CODE REVIEW

File: Source/Common/IGListIndexSetResult.h [hunk 1]
```diff
@@ -43,7 +43,7 @@ NS_ASSUME_NONNULL_BEGIN
 
  @return `YES` if the result has changes, `NO` otherwise.
  */
-- (BOOL)hasChanges;
+@property (nonatomic, assign, readonly) BOOL hasChanges; // Fix For Git Issue #402
```
Comment (rnystrom) [inline Source/Common/IGListIndexSetResult.h]: Can you remove the code comment?

Comment (AdamRobertsEF) [inline Source/Common/IGListIndexSetResult.h]: Yes

**Comment by jessesquires** [comment]
this is breaking

**Comment by AdamRobertsEF** [comment]
@rnystrom I have some github app issue and reforked. I didn't expect it to close the commit, since you already imported.

**Pull request closed without merging**